build.sbt

```diff
@@ -36,39 +36,40 @@ lazy val root = (project in file(".")).enablePlugins(
       "com.gu.memsub.Subscription.ProductRatePlanId"
   ))
 
-scalaVersion := "2.11.8"
-scalacOptions ++= Seq("-feature", "-Xfatal-warnings")
+scalaVersion := "2.11.12"
+scalacOptions ++= Seq("-feature")
```

Comment:
now we have too many warnings? 😞 do you think we can add this back in future?

Reply:
Play throws loads of deprecation warnings. 
:(